.travis.yml

```diff
@@ -1,10 +1,19 @@
 sudo: false
 language: python
-python:
-  - "2.7"
-  - "3.5"
-  - "3.6"
-  - "nightly"
+matrix:
+  include:
+    - python: 2.7
+      dist: trusty
+      sudo: false
```

Comment:
(passing by, interested question) Do you really need to repeat the `sudo: false`, when it's declared at the beginning of the file? Logically, only the 3.7 one would need an override